src/plugins/graph/adornments/plotted-function/plotted-function-adornment-model.ts

```diff
@@ -37,27 +34,42 @@ export const PlottedFunctionAdornmentModel = AdornmentModel
   .props({
     type: types.optional(types.literal(kPlottedFunctionType), kPlottedFunctionType),
     plottedFunctions: types.map(PlottedFunctionInstance),
-    error: ""
+    error: "",
+    xVariableId: types.maybe(types.string),
+    yVariableId: types.maybe(types.string)
```

Comment:
Regarding our discussion about having one adornment with multiple traces, or one adornment model per trace -- noting that this model already has a list of plottedFunctions.  Maybe our x & y variable IDs should go at that "instance" level?